GUI/Types/Renderer/ShaderLoader.cs

```diff
@@ -288,8 +294,8 @@ private uint CalculateShaderCacheHash(string shaderFileName, IDictionary<string,
 
             foreach (var key in parameters)
             {
-                shaderCacheHashString.AppendLine(key);
-                shaderCacheHashString.AppendLine(arguments[key] ? "t" : "f");
+                shaderCacheHashString.Append(key);
+                shaderCacheHashString.Append(Convert.ToBase64String(new[] { arguments[key] }));
```

Comment:
Isn't it a byte, just doing ToString should be enough on it?

Reply:
Murmurhash didn't like numbers, this fixed the collisions. Presumably that's why it used to have "t" and "f".

Reply:
Nevermind ToString works fine now 😄